File: HealthcareApp/healthcare-frontend/src/shared/types/patient/patient.ts
export interface Patient {
  id?: number;
  name: string;
  age: number;
  gender: string;
  bloodType: string;
  address: string;
  testResults?: string;
  createdAt?: string;
}

export type PatientForm = Omit<Patient, 'id' | 'createdAt' | 'testResults'>;

File: HealthcareApp/HealthcareApp.Api/Models/Entities/Patient.cs
using System.ComponentModel.DataAnnotations;

namespace HealthcareApp.Api.Models.Entities
{
    public class Patient
    {
        [Key]
        public int Id { get; set; }

        [Required]
        [MaxLength(100)]
        public string Name { get; set; } = string.Empty;

        public int Age { get; set; }

        [MaxLength(50)]
        public string Gender { get; set; } = string.Empty;

        [MaxLength(200)]
        public string Address { get; set; } = string.Empty;
        public string BloodType { get; set; } = string.Empty;

        public DateTime CreatedAt { get; set; } = DateTime.UtcNow;
        public int HospitalId { get; set; }
    }
}
